README_extending_docker.md

```diff
@@ -137,3 +123,62 @@ You can test the check by running:
  pytest -s actions/<short_name>/test_<short_name>.py
 ```
 Please ensure if your check requires any inputs, you fill the **InputParamsJson** accordingly.
+
+
+
+## Appendix
+
+### Actions Created using Jupyter Lab interface
+
+If you created custom actions using jupyter lab interface and would like to copy the custom actions and runbooks
+to include in your custom docker, then follow this.
+
+#### How to Copy Custom Actions and Runbook 
+
+If you have deployed our Awesome runbook as a Kubernetes POD then follow the step below
+1. Copy the custom actions from the POD to your local machine so you can bundle into your custom Docker for re-distribution
+```
+kubectl cp <AWESOME_POD_NAME>:/unskript/data/runbooks -n <NAMESPACE> $YOUR_REPO_DIRECTORY/runbooks
+kubectl cp <AWESOME_POD_NAME>:/unskript/data/actions -n <NAMESPACE> $YOUR_REPO_DIRECTORY/actions
+cd $YOUR_REPO_DIRECTORY
+git clone https://github.com/unskript/Awesome-CloudOps-Automation.git
+
+Example:
+
+kubectl cp awesome-runbooks-0:/unskript/data/actions -n awesome-ops $YOUR_REPO_DIRECTORY/actions
+kubectl cp awesome-runbooks-0:/unskript/data/runbooks -n awesome-ops $YOUR_REPO_DIRECTORY/runbooks
+```
+
+If you have deployed our Awesome runbook as a Docker instance, then you can use
+the following step.
+```
+export CONTAINER_ID=`docker ps | grep awesome-runbooks | awk '{print $1}'`
+docker cp $CONTAINER_ID:/unskript/data/actions $HOME/Workspace/acme/actions
+docker cp $CONTAINER_ID:/unskript/data/runbooks $HOME/Workspace/acme/runbooks
+```
+
+### Running Actions using unskript-ctl.sh 
+
+If you want to run these actions (or checks) that take arguments via our CLI interface `unskript-ctl.sh` inside the custom docker, then
+follow these steps.
+
+#### How to specify values for arguments
+
+As aluded to earlier actions (or checks) can take one ore more arguments. If you want to run actions that take arguments via the
+`unskript-ctl.sh` CLI, then you need to specify the values for such arguments in the global configuration file named 
+`unskript_config.yaml`.  Inside the awesome-runbooks docker the location would be in `/unskript/data/actions` folder.  
+If you are extending the docker then this file need to be in the  `YOUR_REPO_DIRECTORY/actions/` folder.
+
+Here is how you specific the values for such arguments under the `globals` tag.
+
+   ```
+   globals:
+      namespace: "awesome-ops"
```

Comment:
add some mention of namespace being check argument

Reply:
Done.